vts/trustedservices/trustedservices_grpc.go

```diff
@@ -418,17 +421,23 @@ func (c *GRPC) initEvidenceContext(
 	return appraisal, err
 }
 
-func (c *GRPC) getTrustAnchor(id string) (string, error) {
-	values, err := c.TaStore.Get(id)
-	if err != nil {
-		return "", err
-	}
+func (c *GRPC) getTrustAnchors(id []string) ([]string, error) {
 
-	if len(values) != 1 {
-		return "", fmt.Errorf("found %d trust anchors, want 1", len(values))
+	var taValues []string //nolint
+	for _, taID := range id {
```

Comment:
Same note as above here -- we should add a `GetMultiple` to the store interface.

Reply:
https://github.com/veraison/services/issues/207 raised to track this enhancement!